asynch/__init__.py

```diff
@@ -1,3 +1,3 @@
-from asynch.connection import Connection, connect  # noqa: F401
+from asynch.connection import Connection  # noqa: F401
 from asynch.cursors import Cursor, DictCursor  # noqa: F401
-from asynch.pool import Pool, create_pool  # noqa: F401
+from asynch.pool import Pool  # noqa: F401
```

Comment:
Removing these functions seems to me as a right thing to do. The main reason is their corresponding classes are powerful enough to be used as asynchronous context managers, so no need in duplicating some API.